doc-content/jbpm-docs/src/main/asciidoc/CoreEngine/event-listeners-con.adoc

```diff
@@ -0,0 +1,57 @@
+[id='event-listeners-con_{context}']
+= Event Listeners
+
+You can develoip a class that implements the `ProcessEventListener`. This class can listen to process-related events, like starting or completing a process, entering and leaving a node, and so on.
```

Comment:
I moved event listeners out of the kie base/session section and into their own section like threads, because it turns out they are used a lot when using services.